### DISCUSSION TITLE
AVIF file shows vips warning

### DISCUSSION THREAD

**Discussion by jfcalvo** [libvips/libvips#2734] [Q&A] [answered]
I'm generating an AVIF file from a PDF using the following command:

```
$ vips heifsave sample.pdf[page=0] output.avif --compression av1
```

After that trying to inspect the file with `vipsheader` I get the following warning:

```
$ vipsheader --all output.avif
(vipsheader:13267): VIPS-WARNING **: 20:31:21.261: heifload: ignoring nclx profile
output.avif: 612x792 uchar, 4 bands, srgb, heifload
width: 612
height: 792
bands: 4
format: uchar
...
```

Using libvips 8.2.12 and libheif 1.12.0 on MacOS.

Here the files:
* [sample.pdf](https://github.com/libvips/libvips/files/8344824/sample.pdf)
* [output.avif.zip](https://github.com/libvips/libvips/files/8344832/output.avif.zip)

I have tried with other PDFs with the same result.

Any reason why that warning is raised? I can successfully open the AVIF file on Google Chrome without any problem but I wanted to share the problem anyway.

**Comment by jcupitt** [comment, marked as answer]
Hi @jfcalvo,

Yes, it's safe to ignore in this case. nclx profiles are a type of cut-down colour profile that's becoming popular for video, though libvips does not support them yet.

The AVIF saver is adding an empty profile for this image, then on load libvips is warning that it's ignoring the profile (not knowing that the profile is empty).

**Reply by jfcalvo** [reply]
Understood. Thanks for your answer.

**Comment by kleisauke** [comment]
PR https://github.com/libvips/libvips/pull/3834 downgrades this `g_warning()` to a `g_info()`.
